ultravox/training/train.py

```diff
@@ -303,6 +303,8 @@ def main() -> None:
         # Merge LoRA weights for better inference performance.
         # Note: this is irreversible and changes model saving format
         model.merge_and_unload()
+        # padding_side="left" is required for (batch) inference
+        text_tokenizer.padding_side = "left"
```

Comment:
We use batch_size of 1 and no collator, so this should not be needed.

Reply:
`left` is enforced in the `__init__` method

https://github.com/fixie-ai/ultravox/blob/f2fbdafd5e6eecbd22b70431b3bd7cf5da60d47e/ultravox/inference/infer.py#L44

Reply:
Oh okay. Seems like the assert was wrong to put in then. I was confused too when Pat said `UltravoxInference` does the left padding but enforced `padding_left` in `LocalInference`.

We could also move the assert to the start of the `infer_batch` function.

But no strong opinion.

Reply:
Agreed. On the other hand, I think it is better to keep it consistent for the `LocalInference` in terms of `padding_side`.